eng/native/functions.cmake

```diff
@@ -68,16 +68,8 @@ endfunction(get_include_directories)
 function(get_include_directories_asm IncludeDirectories)
     get_directory_property(dirs INCLUDE_DIRECTORIES)
 
-    if (CLR_CMAKE_HOST_ARCH_ARM AND WIN32)
-        list(APPEND INC_DIRECTORIES "-I ")
-    endif()
-
     foreach(dir IN LISTS dirs)
-      if (CLR_CMAKE_HOST_ARCH_ARM AND WIN32)
-        list(APPEND INC_DIRECTORIES ${dir};)
-      else()
-        list(APPEND INC_DIRECTORIES -I${dir})
-      endif()
+      list(APPEND INC_DIRECTORIES -I${dir};)
```

Comment:
I was running into some issues due to this. I am guessing that at some point arm32 asm did not support more than one include folder option. It looks like it supports now. 

I do not remember exactly the issues with this. I think it was something with properly escaping the list of included directories (which now contain dots since we must match intermediates for OBJECT libraries).
The individually specified multiple "-I{dir}" just worked, so I made arm32 use that as well.